spec/support/shared_examples/core_features.rb

```diff
@@ -32,22 +32,24 @@
 
   if skip_examples.exclude?(:topics)
     describe "Topics" do
-      context "with an anonymous user" do
-        before { visit "/" }
+      if skip_examples.exclude?(:"topics:read")
```

Comment:
At this point... should we just switch to using real strings instead of symbols? `:"foo"` is kinda unusual syntax, and I don't think it's providing any benefit over `"foo"` ?

Reply:
It’s just that using `%i[]` lets you write it as you would expect: `%i[topics:read search:quick_search]`. But I could change it to strings, indeed, I’m just usually sticking to symbols for those kinds of things.

Reply:
It would also break the stuff we already merged, even if there’s not so much at the moment.

Reply:
`%i` could become `%w`, so it would be

```
%w(topics:read search:quick_search)
```

And for backward compat, we could add a `.map(&:to_s)`...

But yeah I guess when I see it as `%i` the symbol pattern is ok. Certainly better than `:"foo"`. Happy to go with this if it's your preference.

Reply:
Ok then, I’ll go with it as is. Thanks! :slightly_smiling_face: